index/scorch/persister.go

```diff
@@ -84,6 +84,15 @@ type notificationChan chan struct{}
 func (s *Scorch) persisterLoop() {
 	defer s.asyncTasks.Done()
```

Comment:
This line can be moved into the defer func below.

Reply:
Thanks, I moved it. I've also find a redundant call of Done below, but now I see it's already fixed in master.